_layouts/default.html

```diff
@@ -19,9 +19,8 @@ <h1><a href="/">Association Agile France</a></h1>
 		<footer>
 			<p>
```

Comment:
L'adresse postale ainsi que l'adresse email devraient être englobées dans un [élément `<address>`](https://www.w3.org/TR/2014/PR-html5-20140916/sections.html#the-address-element) au lieu du `<p>`. Cet élément devrait également avoir un attribut [`translate="no"`](https://www.w3.org/TR/2014/PR-html5-20140916/dom.html#the-translate-attribute) pour en empêcher la traduction automatique.

Reply:
Done with the email but I disagree with the postal address:

> The address element must not be used to represent arbitrary addresses (e.g. postal addresses), unless those addresses are in fact the relevant contact information. (The p element is the appropriate element for marking up postal addresses in general.)

Reply:
Fair enough.